source/beamline/SGM/SGMHexapod.cpp

```diff
@@ -193,3 +273,20 @@ void SGMHexapod::resetSystem()
 	zAxisPrimeTrajectoryControl_->resetRotationsToGlobal();
 }
 
+QVector<QVector3D> SGMHexapod::transformVectors(const QVector<QVector3D> &coordinates)
```

Comment:
Not to be overly pedantic, but this would probably be better as a QList<QVector3D>.  We use QVector a lot for data because it guarantees that the data is aligned in memory, but that isn't necessary for general containers.  In general, QList is the way to go when wanting a container, QVector for a data container, and QStringList for a string container.